tests/RestHandler/VstResponse-test.cpp

```diff
@@ -0,0 +1,75 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2020 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Ignacio Rodriguez
+////////////////////////////////////////////////////////////////////////////////
+
+#include "gtest/gtest.h"
+#include "Rest/VstResponse.h"
+#include "velocypack/Parser.h"
+#include "Basics/encoding.h"
+
+#define ASSERT_ARANGO_OK(x) \
+  { ASSERT_TRUE(x.ok()) << x.errorMessage(); }
+
+// -----------------------------------------------------------------------------
+// --SECTION--                                                 setup / tear-down
+// -----------------------------------------------------------------------------
```

Comment:
this seems to be unused in this test, so I suggest removing it
```suggestion
#define ASSERT_ARANGO_OK(x) \
  { ASSERT_TRUE(x.ok()) << x.errorMessage(); }

// -----------------------------------------------------------------------------
// --SECTION--                                                 setup / tear-down
// -----------------------------------------------------------------------------
```

Reply:
Removed as suggested. Thank you.